lib/openpgp_crypt_symmetric.php

```diff
@@ -188,34 +191,37 @@ public static function getCipher($algo) {
         }
         break;
       case 4:
-        $cipher = new Crypt_Blowfish(Crypt_Blowfish::MODE_CFB);
+        $cipher = new Crypt_Blowfish('cfb');
+        $cipher->setIV(str_repeat(pack('x'), 8));
         $key_bytes = 16;
         $key_block_bytes = 8;
         break;
       case 7:
-        $cipher = new Crypt_AES(Crypt_AES::MODE_CFB);
+        $cipher = new Crypt_AES('cfb');
         $cipher->setKeyLength(128);
+        $cipher->setIV(str_repeat(pack('x'), 16));
         break;
       case 8:
-        $cipher = new Crypt_AES(Crypt_AES::MODE_CFB);
+        $cipher = new Crypt_AES('cfb');
         $cipher->setKeyLength(192);
+        $cipher->setIV(str_repeat(pack('x'), 16));
         break;
       case 9:
-        $cipher = new Crypt_AES(Crypt_AES::MODE_CFB);
+        $cipher = new Crypt_AES('cfb');
         $cipher->setKeyLength(256);
+        $cipher->setIV(str_repeat(pack('x'), 16));
         break;
       case 10:
-        $cipher = new Crypt_Twofish(Crypt_Twofish::MODE_CFB);
-        if(method_exists($cipher, 'setKeyLength')) {
-          $cipher->setKeyLength(256);
```

Comment:
I think it was a  "hack" to get    `key_length` to return  32 , because  later the key  is to one that is actually 128 bits long ,  causing phpseclib3 to fail with

```
1) Decryption::testDecryptTwofish
phpseclib3\Exception\InconsistentSetupException: Key length has already been set to 32 bytes and this key is 16 bytes

/vagrant/openpgp-php/vendor/phpseclib/phpseclib/phpseclib/Crypt/Common/SymmetricKey.php:900
/vagrant/openpgp-php/vendor/phpseclib/phpseclib/phpseclib/Crypt/Twofish.php:433
/vagrant/openpgp-php/lib/openpgp_crypt_symmetric.php:142
/vagrant/openpgp-php/lib/openpgp_crypt_symmetric.php:71
/vagrant/openpgp-php/tests/phpseclib_suite.php:71
/vagrant/openpgp-php/tests/phpseclib_suite.php:9
```